egs2/TEMPLATE/asr1/asr.sh

```diff
@@ -94,6 +94,34 @@ num_splits_asr=1           # Number of splitting for lm corpus.
 
 # Decoding related
 use_k2=false      # Whether to use k2 based decoder
+export is_ctc_decoding=true
```

Comment:
Is it possible to add some of them in the inference config?

Reply:
Good point. 
I will move most of these options to inference config file.
And several of them may remain into asr.sh to construct inference_tag.

Now **_export_** is used because shellcheck consided them as "unused varialbe" and ci fails. Here is an example log:
```
In egs2/librispeech/asr1/asr.sh line 106:
nnlm_weight="1.0"
^---------^ SC2034: nnlm_weight appears unused. Verify use (or export if used externally).
```
Though they are definitely used in following **_for loop_** to [construct inference_tag](https://github.com/espnet/espnet/blob/2e8c6cdad4cff64ae849a8af21d99099360490f5/egs2/TEMPLATE/asr1/asr.sh#L440).
```
    if "${use_k2}"; then
      inference_tag+="_use_k2"
      for flag in ${inference_tag_flags_k2}; do
        inference_tag+="_${flag}_${!flag}"
      done
    fi
```

Reply:
fixed by add conf/decode_asr_transformer_with_k2.yaml.

Results are also added into README.md